### PR TITLE
Admin List

### PR DESCRIPTION
**Pull request by tobiasschweizer**
Integrates https://github.com/dasch-swiss/knora-api/pull/1509

### CODE REVIEW

File: src/api/admin/permissions/permissions-endpoint.spec.ts [hunk 1]
```diff
@@ -26,9 +26,38 @@ describe("PermissionsEndpoint", () => {
...
+        });
+
+        it("should return the administrative permissions by type", done => {
+
+            const projectIri = "http://rdfh.ch/projects/00FF";
+
+            const groupIri = "http://www.knora.org/ontology/knora-admin#ProjectMember";
 
-            knoraApiConnection.admin.permissionsEndpoint.getAdministrativePermission(projectIri, groupIri, permissionType).subscribe(
+            knoraApiConnection.admin.permissionsEndpoint.getAdministrativePermissionByType(projectIri, groupIri, "administrative_permissions").subscribe(
```
Comment (tobiasschweizer) [inline src/api/admin/permissions/permissions-endpoint.spec.ts]: @benjamingeer Is this usage correct?

Comment (benjamingeer) [reply src/api/admin/permissions/permissions-endpoint.spec.ts]: I've just noticed that the `permissionType` parameter is actually ignored:

https://github.com/dasch-swiss/knora-api/blob/develop/webapi/src/main/scala/org/knora/webapi/routing/admin/PermissionsRouteADM.scala#L91

So I guess I should comment out the `getAdministrativePermissionByType` function.

Comment (tobiasschweizer) [reply src/api/admin/permissions/permissions-endpoint.spec.ts]: That would make the test pretty useless ;-)

Comment (tobiasschweizer) [reply src/api/admin/permissions/permissions-endpoint.spec.ts]: Shall I remove the test for `getAdministrativePermissionByType` I have just added?

Comment (benjamingeer) [reply src/api/admin/permissions/permissions-endpoint.spec.ts]: I've just commented out the function, so you can remove the test.

Comment (tobiasschweizer) [reply src/api/admin/permissions/permissions-endpoint.spec.ts]: lovely :-)

**Comment by tobiasschweizer** [comment]
@lrosenth Can you use this in MLS app?

**Pull request merged**